site/docs/v1/tech/events-webhooks/writing-a-webhook.adoc

```diff
@@ -18,6 +18,23 @@ Your Webhook must handle the RESTful request described above and send back an ap
 
 Once your Webhook is complete and listening for events, you must configure your Webhook URL in FusionAuth. To add a webhook navigate to [breadcrumb]#Settings -> Webhooks#. If you have multiple Webhooks configured for a single Application, the transaction setting for the event or the User Action will dictate if FusionAuth will commit the transaction or not.
 
+=== Handling Create Events
+
+There are some events fired on creation of an entity in FusionAuth, such as `user.create`. Sometimes you want to modify the created entity, but if your webhook tries to modify the newly created object in a webhook handling the create event, the operation will fail. This is due to the fact that creation occurs in a transaction and has not yet completed when the webhook is running. In fact, it cannot until the webhook returns. The webhook is trying to modify an object that doesn't fully exist, and that operation fails.
```

Comment:
This last sentence is confusing to me.

> In fact, it cannot until the webhook returns. The webhook is trying to modify an object that doesn't fully exist, and that operation fails.

Maybe something like:

> In fact, the user will not be visible to another API request until the transaction is committed. The operation fails because the webhook is trying to modify an object that has not yet completed creation and has not yet been committed to persistant storage. 
>
> Depending upon your transaction configuration for a particular event, FusionAuth may wait until all webhooks have responded before committing the transaction. 
> 
> Even if you configure your webhook transaction to not require any webhooks to succeed, it is unlikely your webhook will succeed due to the parallel timing of the requests. The `user.create` event was not designed to allow a webhook to retrieve and modify the user.